Automattic/wp-calypso
# Fix: Do not show annual discount on the checkout page

#### Changes proposed in this Pull Request

* `Annual Discount` can confuse the users. This PR removes it and brings back "Discount for first `[year/month/term]`" instead.
* "Plan Subscription" also comes back underneath "WordPress.com `[Plan name]`".
* "Plan Subscription" is followed by the original price.

For more details, see pcbrnV-1gA-p2#comment-2768

Screenshots can be found here: pcbrnV-1gA-p2#comment-2847

#### Testing instructions



* Upgrade an annual paid plan and confirm the "Discount for first year" messaging shows.
* Change the term to "One month" and confirm "Discount for first month" shows.
* Change the term to "Two years" and confirm "Discount for first term" appears.
* Add a plan to a free site and confirm this messaging doesn't appear.
